plugins/woocommerce/client/admin/client/settings-payments/components/offline-payment-gateways/offline-payment-gateways.tsx

```diff
@@ -1,92 +1,112 @@
 /**
  * External dependencies
  */
-import { List } from '@woocommerce/components';
 import { PaymentGateway } from '@woocommerce/data';
 import { Card, CardHeader } from '@wordpress/components';
-import React from '@wordpress/element';
+import React, { useEffect, useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import { getAdminSetting } from '~/utils/admin-settings';
-import { OfflinePaymentGatewayListItem } from '../offline-gateway-list-item';
+import {
+	OfflinePaymentGateway,
+	OfflinePaymentGatewayList,
+} from '../offline-gateway-list-item/offline-payment-gateway-list-item';
+import { parseOrdering } from '~/settings-payments/utils';
 
 const assetUrl = getAdminSetting( 'wcAdminAssetUrl' );
 
 interface OfflinePaymentGatewaysProps {
 	registeredPaymentGateways: PaymentGateway[];
+	ordering: Record< string, number >;
+	updateOrdering: ( ordering: Record< string, number > ) => void;
 }
 
 export const OfflinePaymentGateways = ( {
 	registeredPaymentGateways,
+	ordering,
+	updateOrdering,
 }: OfflinePaymentGatewaysProps ) => {
-	// Mock payment gateways for now.
-	const mockOfflinePaymentGateways = [
-		{
-			id: 'bacs',
-			title: __( 'Direct bank transfer', 'woocommerce' ),
-			content: __(
-				'Accept payments via Bacs — more commonly known as direct/wire transfer.',
-				'woocommerce'
-			),
-			image: assetUrl + '/payment_methods/bacs.svg',
-			square_image: assetUrl + '/payment_methods/bacs.svg',
-			image_72x72: assetUrl + '/payment_methods/bacs.svg',
-			actionText: '',
-		},
-		{
-			id: 'cheque',
-			title: __( 'Check payments', 'woocommerce' ),
-			content: __(
-				'Take payments in person via checks. This payment method can also be used for testing purposes.',
-				'woocommerce'
-			),
-			image: assetUrl + '/payment_methods/cheque.svg',
-			square_image: assetUrl + '/payment_methods/cheque.svg',
-			image_72x72: assetUrl + '/payment_methods/cheque.svg',
-			actionText: '',
-		},
-		{
-			id: 'cod',
-			title: __( 'Cash on delivery', 'woocommerce' ),
-			content: __(
-				'Have your customers pay with cash (or by other means) upon delivery.',
-				'woocommerce'
-			),
-			image: assetUrl + '/payment_methods/cod.svg',
-			square_image: assetUrl + '/payment_methods/cod.svg',
-			image_72x72: assetUrl + '/payment_methods/cod.svg',
-			actionText: '',
-		},
-	];
+	const [ offlineGateways, setOfflineGateways ] = useState<
+		OfflinePaymentGateway[]
+	>( [] );
 
-	const availableOfflineGateways = mockOfflinePaymentGateways
-		.map( ( gateway ) => {
-			const installedGateway = registeredPaymentGateways.find(
-				( g ) => g.id === gateway.id
-			);
+	useEffect( () => {
```

Comment:
You're right. However, it seems a rebase is not ideal at this point since there's some trunk stuff included here that's not included in the other branch and will contain some unrelated code in review. 

I'd opt to merge this after the other branch is merged to trunk.

Reply:
Sure thing!

Reply:
> I'd opt to merge this after the other branch is merged to trunk.
@ilyasfoo okay, sounds good, I'll address the PR feedback and merge it!